components/tyk-dev-portal/values.yaml

```diff
@@ -225,3 +225,6 @@ livenessProbe:
 readinessProbe:
   # initialDelaySeconds specifies the seconds used in readiness probe as initial delay.
   initialDelaySeconds: 60
+
+# Service account should be manually created
```

Comment:
Please align the comments with those in the other `serviceAccountName` fields in the rest of the charts.